### PR TITLE
Fix arithmetic error & update `Lessons.get/3`

### LINKED ISSUE #13: Add a 404 page for missing lessons with a translation CTA

### PR DESCRIPTION
**Pull request by doomspork**
To allow us to differentiate between truly missing pages and missing translations, we need `Lessons.get/3` to return more than `nil` for
missing lessons.

With these new results, we can setup our FallbackController to handle the true 404 and translation 404 pages easier.

Closes #13, #26, and #28

### CODE REVIEW

File: lib/school_house_web/templates/error/404.html.eex [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (doomspork) [inline lib/school_house_web/templates/error/404.html.eex:1]: We can most likely make this `error.html` for all generic error messages and rely on the view itself to provide the specific message.

File: lib/school_house_web/views/error_view.ex [hunk 2]
```diff
@@ -3,9 +3,13 @@ defmodule SchoolHouseWeb.ErrorView do
...
-  # def render("500.html", _assigns) do
-  #   "Internal Server Error"
-  # end
+  def render("404.html", %{translation_not_found: true}) do
+    "Translation Not Found — CTA here"
+  end
+
+  def render("404.html", assigns) do
+    "Not Found"
+  end
```
Comment (doomspork) [inline lib/school_house_web/views/error_view.ex:12]: I will expand on these and setup gettext :+1:

File: lib/school_house_web/templates/error/translation_missing.html.eex [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="bg-white">
+  <div class="max-w-2xl px-4 py-16 mx-auto text-center sm:py-20 sm:px-6 lg:px-8">
+    <h2 class="text-3xl font-extrabold text-brand-purple sm:text-4xl">
+      <span class="block">Translation unavailable</span>
```
Comment (doomspork) [inline lib/school_house_web/templates/error/translation_missing.html.eex:4]: I'm going to wait on the gettext stuff for right now. Then open up some issues and work through it in a separate PR.

**Review by gemantzu — APPROVED** [review]
:fire:

**Pull request merged**